### ISSUE TITLE
[FEAT]: leaving modus "review flashcards" and directly jump to note

### ISSUE THREAD

**Issue by SeLiEnisyst** [st3v3nmw/obsidian-spaced-repetition#786]
Hi,
during my spaced repetition review, I frequently feel the urge, to directly jump to the note, in which I have the flashcard-tag and my flashcard-content. 
As until now it is not possible to directly jump to the relevant note, I first have to live the review, than change the note and then scroll to the part of the note, where my flashcard-tag is located...
=> it would be nice, to have the possibility to directly jump to that part of note.
kind regards,
Silias

**Comment by ronzulu** [comment]
Nice idea, I would find it useful as well.

**Comment by arias007** [comment]
It's a good idea, add a button of jump to file

**Comment by Newdea** [comment]
Versions from long ago had this button to open note, but it's not clear why later versions don't have it anymore.

**Comment by ronzulu** [comment]
> Versions from long ago had this button to open note, but it's not clear why later versions don't have it anymore.

I'm not sure, but perhaps when the flashcard modal was enhanced to include the "edit" button, it was left off.

But even with the edit button, which I find useful, being able to open the note would also be helpful. I guess another button to add here

![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/75528127/68946651-63b4-46b6-8506-fd21487e93fd)

Cheers
Ronny

**Comment by pikatwinky** [comment]
Yes in previous versions there was a button called "Edit Later" which opened the current card's file in a new tab and the cursor moved to its contents. 
I find the current "Edit" button very useful for quick edits, but being able to edit in the current file would allow us to leverage the use of plugins like "Various Complements", "Latex Suite", "Template(r)"...or vim, which allow to ease any complex edit.

I wish the "Edit Later" capability be incorporated in the current "Edit" button as a Ctrl+click shortut in desktop, and a long press on mobile, as some buttons found in the excalidraw plugin does
![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/31860235/44325878-40d8-4ea0-940a-d50c92928510)

**Comment by ronzulu** [comment]
And #661

**Comment by KatzMeIfYouCan** [comment]
> Would be nice to have it back. I was first satisfied that I'd be able to do this when watching this video:
> <img alt="Image" width="790" src="https://private-user-images.githubusercontent.com/366038/406661022-d6119823-c989-4e64-bf3e-e009dd0d54cb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDAzNDQ2MzIsIm5iZiI6MTc0MDM0NDMzMiwicGF0aCI6Ii8zNjYwMzgvNDA2NjYxMDIyLWQ2MTE5ODIzLWM5ODktNGU2NC1iZjNlLWUwMDlkZDBkNTRjYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIyM1QyMDU4NTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00YzNkMzVkYjZiZjgxMzkyNDdmMzJkZDY4NWQwYjczZjY4Y2RiYWJiNjdmNWM2M2U0YzdjZWQ4Y2ZmZjNhMjU4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.29bdijN4Iz8jRxoISXx4GiyLDAYym-E7hlOL-BEOkJg">
> 
> [Learning New Things Using Obsidian](https://youtu.be/DwSNZEW6jCU?t=521)
> 
> I have one note per word which declares the flash card and also contains a lot of additional info -- more than my flash card answer. As in the video, being able to go and see the whole note is something I'd might like to do, especially if I can't remember the answer.

I watched the same video as well as a couple of others to try and figure out why I wasn't able to directly link to the card. I thought I was going crazy! 

This is one of the biggest value props to me, and is very different than the 'quick edit' option we have now, as many plugins are leveraged between actual pages vs. the simple markdown version of the quick edit. I'd love to see this brought back in some shape or form.

**Comment by KyleKlus** [comment]
Hi, so this is properly implemented now😊
